src/Components/PieBarContainer.jsx

```diff
@@ -0,0 +1,140 @@
+import { Flex, Text } from "@sparrowengg/twigs-react";
+import AlertIcon from "./../asset/alert-circle.svg";
+import AlertIcon2 from "./../asset/alert-circle2.svg";
+import PieChartComp from "./PieChartComp";
+import PieChartInfo from "./PieChartInfo";
+import CustomProgressBar from "./CustomProgressBar";
+import ProgressBarInfo from "./ProgressBarInfo";
+import { LightMode } from "../redux/light-dark/lightDarkTypes";
+import { useSelector } from "react-redux";
+import SoldUnsold from "../hooks/SoldUnsold";
+import TopCateogry from "../hooks/TopCateogry";
+import { COLORS } from "../utils/data";
+
+const PieBarContainer = () => {
+  const mode = useSelector((store) => store.lightdarkmode.lightDarkMode);
+
+  const { pieData } = TopCateogry();
+  const { soldUnsoldData } = SoldUnsold();
+
+  return (
+    <Flex
+      css={{
+        marginTop: "25px",
+        width: "100%",
+        height: "100%",
+      }}
+      gap="60px"
+      alignItems="flex-start"
+    >
+      {/* Pia container */}
+      <Flex flexDirection="column" justifyContent="flex-start">
+        <Flex alignItems="center" gap="10px">
+          <Text
+            css={{
+              fontSize: "$md",
+              fontWeight: "700",
+              lineHeight: "28px",
+              color: "$textPrimary",
+            }}
+          >
+            Perpetual
+          </Text>
+          <img src={mode === LightMode ? AlertIcon : AlertIcon2} alt="" />
+        </Flex>
+
+        <Flex
+          css={{
+            marginLeft: "-30px",
+          }}
+          gap="40px"
+          justifyContent="flex-start"
+          alignItems="center"
+        >
+          <PieChartComp arr={pieData?.arr} />
+          <Flex flexDirection="column" gap="28px">
+            {pieData?.arr.map((info, idx) => {
+              return (
+                <PieChartInfo
+                  color={COLORS[idx]}
+                  key={info.title}
+                  title={info[0]}
+                  value={info[1]}
```

Comment:
@9xVibee how is the data presented here, are you only using value from 2nd's info?

Reply:
yess!